data/google-analytics.json

```diff
@@ -19,8 +19,7 @@
       "params": ["id"],
       "optionalParams": {
         "l": "dataLayer",
-        "consentType": "default",
-        "consentValues": null
```

Comment:
See https://github.com/GoogleChromeLabs/third-party-capital/pull/72#discussion_r1759740734 - while technically it doesn't matter, I think there is value in keeping parameters explicit in the schema, even if their default is null or undefined.

Reply:
I understand now, let's add it back :)